README_zh-CN.md

```diff
@@ -2,35 +2,35 @@
   <img src="https://github.com/open-mmlab/mmaction2/raw/main/resources/mmaction2_logo.png" width="600"/>
   <div>&nbsp;</div>
   <div align="center">
-    <b><font size="5">OpenMMLab 官网</font></b>
+    <b><font size="5">OpenMMLab官网</font></b>
     <sup>
       <a href="https://openmmlab.com">
-        <i><font size="4">HOT</font></i>
+        <i><font size="4">火</font></i>
       </a>
     </sup>
     &nbsp;&nbsp;&nbsp;&nbsp;
-    <b><font size="5">OpenMMLab 开放平台</font></b>
+    <b><font size="5">OpenMMLab平台</font></b>
     <sup>
       <a href="https://platform.openmmlab.com">
-        <i><font size="4">TRY IT OUT</font></i>
+        <i><font size="4">尝试一下</font></i>
       </a>
     </sup>
   </div>
 
 [![Documentation](https://readthedocs.org/projects/mmaction2/badge/?version=latest)](https://mmaction2.readthedocs.io/en/latest/)
 [![actions](https://github.com/open-mmlab/mmaction2/workflows/build/badge.svg)](https://github.com/open-mmlab/mmaction2/actions)
-[![codecov](https://codecov.io/gh/open-mmlab/mmaction2/branch/master/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmaction2)
+[![codecov](https://codecov.io/gh/open-mmlab/mmaction2/branch/main/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmaction2)
 [![PyPI](https://img.shields.io/pypi/v/mmaction2)](https://pypi.org/project/mmaction2/)
-[![LICENSE](https://img.shields.io/github/license/open-mmlab/mmaction2.svg)](https://github.com/open-mmlab/mmaction2/blob/master/LICENSE)
+[![LICENSE](https://img.shields.io/github/license/open-mmlab/mmaction2.svg)](https://github.com/open-mmlab/mmaction2/blob/main/LICENSE)
 [![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/open-mmlab/mmaction2.svg)](https://github.com/open-mmlab/mmaction2/issues)
 [![Percentage of issues still open](https://isitmaintained.com/badge/open/open-mmlab/mmaction2.svg)](https://github.com/open-mmlab/mmaction2/issues)
 
-[📘文档](https://mmaction2.readthedocs.io/zh_CN/latest/) |
-[🛠️安装指南](https://mmaction2.readthedocs.io/zh_CN/latest/get_started.html) |
-[👀模型库](https://mmaction2.readthedocs.io/zh_CN/latest/modelzoo.html) |
-[🆕更新](https://mmaction2.readthedocs.io/zh_CN/latest/notes/changelog.html) |
-[🚀进行中项目](https://github.com/open-mmlab/mmaction2/projects) |
-[🤔问题反馈](https://github.com/open-mmlab/mmaction2/issues/new/choose)
+[📘文档](https://mmaction2.readthedocs.io/en/latest/) |
+[🛠️安装](https://mmaction2.readthedocs.io/en/latest/get_started/installation.html) |
+[👀模型库](https://mmaction2.readthedocs.io/en/latest/model_zoo/modelzoo.html) |
+[🆕更新日志](https://mmaction2.readthedocs.io/en/latest/notes/changelog.html) |
+[🚀正在进行的项目](https://github.com/open-mmlab/mmaction2/projects) |
+[🤔报告问题](https://github.com/open-mmlab/mmaction2/issues/new/choose)
```

Comment:
We would like to keep these links to Chinese documentation.

Reply:
Links updated to Chinese version